modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -446,9 +446,17 @@ def mask(
 
         Notes
         -----
-        If both `row_labels` and `row_positions` are set, `row_labels` will be used.
-        The same rule applied to `col_labels` and `col_positions`.
+        If both row_labels and row_positions are provided, a ValueError is raised.
+        The same rule applies for col_labels and col_positions.
```

Comment:
```suggestion
        If both `row_labels` and `row_positions` are provided, a ``ValueError`` is raised.
        The same rule applies for `col_labels` and `col_positions`.
```

Why parameter highlighting was removed?